### DISCUSSION TITLE
Embedded WebServer for LAN

### DISCUSSION THREAD

**Discussion by seri3s** [zim-desktop-wiki/zim-desktop-wiki#2506] [Q&A]
Hi,

Zim is accessed  as http://localhost:8080 starting Zim embedded webServer (ZIM >Tools＞Start Web Server).
It works fine.

How to expand accessing for private LAN?
Where is the conf file for Zim embedded webServer?
Is there a listening parameter such as 192.168.1.0/24

Best Regards,
Shigeru

**Comment by dehesselle** [comment]
No, it's not possible at the moment, `localhost` is hard-coded. The feature for a public webserver is there (in an unfinished state), but it's marked with todos and commented out.

If you run a ssh server on your machine, you can create a tunnel as workaround to forward a port from your public interface to the localhost one.

Or you can just edit the code where it says `localhost` and put your IP in there.
